### ISSUE TITLE
Add Coinbasepro stablecoin conversions

### ISSUE THREAD

**Issue by amarvin** [ccxt/ccxt#4419]
Would be nice to add the coinbasepro API endpoint `/conversions` (https://docs.pro.coinbase.com/?r=1#stablecoin-conversions) for creating a stablecoin conversion (e.g. USD to USDC) to ccxt, as many coinbasepro altcoin products only trade against USDC.

- OS: Windows 10
- Programming Language version: Python 3.6.3
- CCXT version: 1.18.61
- Exchange: coinbasepro
- Method: conversions

**Comment by amarvin** [comment]
Shouldn't a function also be included or maybe include stablecoin conversions as a zero-fee tradeable product?

**Comment by kroitor** [comment]
@amarvin 

> Shouldn't a function also be included or maybe include stablecoin conversions as a zero-fee tradeable product?

Well... it is included now in 1.18.93+. 

- https://github.com/ccxt/ccxt/wiki/Manual#implicit-api-methods

```Python
import ccxt
from pprint import pprint
exchange = ccxt.coinbasepro({
    'enableRateLimit': True,  # required by the Manual
    'apiKey': 'YOUR_API_KEY',
    'secret': 'YOUR_SECRET',
})
exchange.load_markets()
params = {
    'from': exchange.currencies['ETH']['id'],
    'to': exchange.currencies['BTC']['id'],
    'amount': "123.45",
}
response = exchange.private_post_conversions(params)
pprint(response)
```

Let us know if that does not answer the question.
